src/uu/chroot/src/chroot.rs

```diff
@@ -9,12 +9,13 @@ mod error;
 use crate::error::ChrootError;
 use clap::{Arg, ArgAction, Command};
 use std::ffi::CString;
-use std::io::Error;
+use std::io::{Error, ErrorKind};
 use std::os::unix::prelude::OsStrExt;
+use std::os::unix::process::CommandExt;
 use std::path::{Path, PathBuf};
-use std::process;
+use std::process::Command as ProcessCommand;
```

Comment:
I think it's better to not mix cosmetic changes (`ProcessCommand` instead of `process::Command`) with functional fixes.